lib/jxl/decode.cc

```diff
@@ -2701,6 +2703,7 @@ JxlDecoderStatus JxlDecoderGetFrameHeader(const JxlDecoder* dec,
     return JXL_API_ERROR("no frame header available");
   }
   const auto& metadata = dec->metadata.m;
+  memset(header, 0, sizeof(*header));
```

Comment:
I believe the following would be cleaner:
...
  // You may want to initialize `frame_header` here since not all fields are
  // guarantee to be initialized in call to `JxlDecoderGetFrameName`:
  JxlEncoderInitFrameHeader(&frame_header);
...

Reply:
Ok nevermind, this function is defined in `#include "jxl/encode.h"`